Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
Hah, wow.  I am fully going to blame this on baby brain from lack of decent sleep.  After many attempts at telling me that you changed it so it WON'T dispose, I _still_ saw it incorrectly as you removed `false` from the line (I swear I'm not dyslexic either).

I see it correctly now.  

This being said, the change you made would only make a difference if you call `e.Graphics.Dispose()` from Eto's `Drawable.Paint` event which you should not ever be doing anyway, so the change is (should be?) moot.

Again, I apologize for misunderstanding the change you made in the first place!